code/global.dm

```diff
@@ -783,3 +784,24 @@ var/global/mentorhelp_text_color = "#CC0066"
 		globalImages[key] = null
 		globalImages.Remove(key)
 	return
+
+/proc/addAIImage(var/image/I, var/key)
+	if(I && length(key))
+		aiImages[key] = I
+		for(var/mob/M in AIs)
+			if (M.client)
+				M << I
+		return I
+	return
+
+/proc/getAIImage(var/key)
```

Comment:
Not called?

Reply:
i copied the way GlobalImage procs worked, and this *might* get used for similar purposes later if we want to draw things that arent static and make changes to em at runtime